pkgs/by-name/cl/clevis/package.nix

```diff
@@ -105,11 +105,9 @@ stdenv.mkDerivation (finalAttrs: {
       clevisZfsFallback
       ;
     clevisLuksSystemdStage1 = nixosTests.installer-systemd-stage-1.clevisLuks;
-    clevisLuksFallbackSystemdStage1 =
-      nixosTests.installer-systemd-stage-1.clevisLuksFallback;
+    clevisLuksFallbackSystemdStage1 = nixosTests.installer-systemd-stage-1.clevisLuksFallback;
```

Comment:
Why is the commit `clevis: nixfmt-rfc-style (without line limits)` not squashed with it's parent?